python/lsst/ip/isr/isrTask.py

```diff
@@ -788,11 +1035,28 @@ def run(self, ccdExposure, camera=None, bias=None, linearizer=None, crosstalkSou
         exposure after all ISR processing has finished.
 
         """
-        if isinstance(ccdExposure, ButlerDataRef):
-            return self.runDataRef(ccdExposure)
 
-        self.log.info("Performing ISR on exposure %s" %
-                      (ccdExposure.getInfo().getVisitInfo().getExposureId()))
+        if isGen3 is True:
+            # Gen3 currently cannot do configuration overrides.  Manually do so,
+            # supplying it with the obs_subaru configuration for HSC.
+            # DM-15257 looks to discuss this issue.
+
+            obsConfigDir = os.path.join(getPackageDir("obs_subaru"), "config", "hsc")
```

Comment:
Same concern as above that we are hard coding config overrides.  Thanks for including the ticket for discussion here.